libcfnet/server_code.c

```diff
@@ -192,20 +195,32 @@ static int OpenReceiverChannel(char *bind_address)
  */
 int InitServer(size_t queue_size, char *bind_address)
 {
-    int sd = OpenReceiverChannel(bind_address);
-
-    if (sd == -1)
-    {
-        Log(LOG_LEVEL_ERR, "Unable to start server");
-    }
-    else if (listen(sd, queue_size) == -1)
+    #if HAVE_SYSTEMD_SD_DAEMON_H
```

Comment:
Please move the preprocessor directives to the beginning of the line (with no indentation).